packages/office-addin-cli/tsconfig.json

```diff
@@ -3,6 +3,7 @@
         "target": "es6",
         "module": "commonjs",
         "moduleResolution": "node",
+        "esModuleInterop": true,
```

Comment:
What does this config option do?

Reply:
See https://www.typescriptlang.org/tsconfig#esModuleInterop. This isn't required as part of these changes, but it is better if this is set to true. 

With TypeScript code, you shouldn't use `const package = require("package")`. There are a few forms of import statements that can apply depending on how the package exports its modules. You can read about modules here: https://www.typescriptlang.org/docs/handbook/modules.html.